sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -398,3 +398,33 @@
   height: 100%;
   min-height: $image-min-height;
 }
+
+.muted-content {
+  color: #69768c;
```

Comment:
We do not have a constant for this color?

Reply:
mm,  actually this color was not in the global variable

Reply:
created  a global variable for this color